doc/discretization.rst

```diff
@@ -0,0 +1,282 @@
+=======================
+Discretization Strategy
+=======================
+
+.. _disc-strat:
+
+Literature on DG for Fluid Flow
```

Comment:
I feel like this would be more readable if it also got the `\let\b=\mathbf` treatment. (Just copy from the other file.)

Reply:
I am really struggling with this. Copy *what* from *what* other file? I searched meshmode, grudge, mirgecom, and google for how to do this, and came up empty handed.

I tried adding `latex_elements = {"preamble": r"""\let\b=\mathbf"""}`, to no avail. It does not appear to render properly.

Reply:
(f1a7248) :tada:?

Reply:
I vote we not let this hold up the PR unnecessarily and give the latex compiler an excuse to bug out on us.

Reply:
#356